### ISSUE TITLE
Search -- format to get listing info from AirBnB API

### ISSUE THREAD

**Issue by penguyen1** [penguyen1/Padticular#39]
###### Provided the unique AirBnB id, the format to get a listing's info from the AirBnB API should look like:
- **NOTE TO SELF:**
  - only needs the api key & AirBnB listing id!
  - the returned response has twice as many ( & more specific ) information for the user

``` javascript
// AirBnB_listing_id will be provided prior
var AirBnB_key = < AirBnB API key >;   // will be hidden
var URL = 'https://api.airbnb.com/v2/listings/${AirBnB_listing_id}?client_id=${AirBnB_key}&locale=en-US&currency=USD&_format=v1_legacy_for_p3';
```

**Comment by ahmadhashlamoun** [comment]
I'm tryin to scrap Airbnb listing information page, I used to request it like so:

```https://api.airbnb.com/v2/listings/5116458?client_id=3092nxybyb0otqw18e8nh5nty&_format=v1_legacy_for_p3```

But now they return the format **_format=v1_legacy_for_p3** is no longer supported, could anybody advice how could I get the new supported one to get a listing information?

@penguyen1 @jasonseminara @pppetrov

**Comment by joshbagnallcom** [comment]
Not sure if this is relevant to you still @ahmadhashlamoun but I was recently having the same exact issue with a site I built for a client several years ago.

After about an hour of messing around with it, I found this works:
https://api.airbnb.com/v2/listings/?client_id=CLIENT_ID&listing_ids=LISTING_ID

It works but doesn't return amenities as it seemed to before.

I got there by deleting the listing id from the call which returned error details stating "At least one of these rules must be satisfied: (current_user_listing is required.) OR (user_id is required.) OR (top_listing is required.) OR (recently_viewed_listing is required.) OR (recently_viewed_with_similar_listings is required.) OR (compare_listing is required.) OR (listing_ids is required.) OR (recommended_listing is required.)".

I am still trying to retrieve the amenities...
